assets/src/edit-story/app/media/useUploadMedia/useUploadMedia.js

```diff
@@ -87,10 +87,12 @@ function useUploadMedia({ media, pagingNum, mediaType, fetchMedia, setMedia }) {
             sprintf(
               /* translators: %s: list of allowed file types. */
               __('Please choose only <b>%s</b> to upload.', 'web-stories'),
-              allowedMimeTypes.join(
-                /* translators: delimiter used in a list */
-                __(', ', 'web-stories')
-              )
+              allowedMimeTypes
+                .map((type) => type.replace('image/', '').replace('video/', ''))
```

Comment:
Should this be extracted and used in all places? Should there be tests?
What about instead just splitting the mime type and using the second part if any?

Reply:
Indeed looks like a great candidate to extract and add a quick unit test for it. Perhaps as its own `useAllowedFileTypes` hook in `assets/src/edit-story/utils`.

Should be quick to write with `split` and using the second part, as you suggest. We could probably even prefix the result with a dot so `image/png` gets turned into `.png`

Also, the way `allowedMimeTypes` is assembled throughout is different:

```js
const { allowedMimeTypes = {} } = useConfig();
const allowedFileTypes = Object.keys(allowedMimeTypes).reduce(
  (acc, cur) => [...acc, ...allowedMimeTypes[cur]],
  []
);

// vs.

const {
  allowedMimeTypes: {
    image: allowedImageMimeTypes,
    video: allowedVideoMimeTypes,
  },
} = useConfig();
const allowedMimeTypes = [...allowedImageMimeTypes, ...allowedVideoMimeTypes];
```

The first approach seems more future proof.

So the result could look something like (untested):

```js
function useAllowedFileTypes() {
  const { allowedMimeTypes = {} } = useConfig();
  const allowedFileTypes = Object.keys(allowedMimeTypes).reduce(
    (acc, cur) => [...acc, ...allowedMimeTypes[cur]],
    []
  );

  allowedMimeTypes
    .map((type) => '.' + type.split('/')[1])
    .join(
      /* translators: delimiter used in a list */
      __(', ', 'web-stories')
     );

  return allowedMimeTypes;
}
```

Reply:
Using allowedFileTypes instead.